File: servico_basico/src/java/modelo/Departamento.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package modelo;

/**
 *
 * @author superalunocmc
 */
public class Departamento extends Objeto {

    private String descricao;

    public Departamento() {
    }

    public Departamento(String descricao, int id) {
        super(id);
        this.descricao = descricao;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
}

File: Cadastro de Pessoa/Cadastro de Pessoa/Modelo/Departamento.cs
namespace ClienteREST.Modelo
{
    public class Departamento:Objeto
    {
        public string descricao { get; set; }
        public Departamento() { }
        public Departamento(int id, string descricao)
        {
            this.id = id;
            this.descricao = descricao;
        }
    }
}
